docs/sources/operators-guide/running-production-environment/production-tips/index.md

```diff
@@ -30,6 +30,13 @@ The required disk space depends on the number of time series stored in the inges
 
 For more information about estimating the required ingester disk space, refer to [Planning capacity]({{< relref "../planning-capacity.md#ingester" >}}).
 
+### Ingester disk IOPS
+
+The ingester disk performances (IOPS and latency) can impact both write and read requests.
```

Comment:
```suggestion
The ingester disk performances (IOPS and latency) can affect both write and read requests.
```
The first definition for _impact_ is a crash or collision; the second definition means what you intend here.

Reply:
Please vet this for technical accuracy.

Reply:
Also, I assume "disks" is plural here.